beacon-chain/core/electra/consolidations.go

```diff
@@ -206,14 +209,16 @@ func ProcessConsolidationRequests(ctx context.Context, st state.BeaconState, req
 		}
 
 		if npc, err := st.NumPendingConsolidations(); err != nil {
-			return fmt.Errorf("failed to fetch number of pending consolidations: %w", err) // This should never happen.
+			log.WithError(err).Errorf("failed to fetch number of pending consolidations at index %d", i)
+			continue
 		} else if npc >= pcLimit {
 			return nil
 		}
 
 		activeBal, err := helpers.TotalActiveBalance(st)
 		if err != nil {
-			return err
+			log.WithError(err).Error("failed to fetch total active balance")
```

Comment:
I think for these kinds of errors the request may be valid and we fail to get the active balance for whatever reason, by continuing we would have a consensus split anyway since the state root will fail to match. I think we need to return an error and not mark the block as invalid though.